quantus/metrics/robustness/relative_output_stability.py

```diff
@@ -0,0 +1,314 @@
+    to the change in the output logits
+
+    :math:`ROS(x, x', ex, ex') = max \\frac{||\\frac{e_x - e_x'}{e_x}||_p}{max (||h(x) - h(x')||_p, \epsilon_{min})}`,
+
+    where `h(x)` and `h(x')` are the output logits for `x` and `x'` respectively
+
+
+    References:
+        1) Chirag Agarwal, et. al., 2022. "Rethinking stability for attribution based explanations.", https://arxiv.org/pdf/2203.06877.pdf
+    """
+
+    @attributes_check
+    def __init__(
+        self,
+        nr_samples: int = 200,
+        abs: bool = False,
+        normalise: bool = False,
+        normalise_func: Optional[Callable[[np.ndarray], np.ndarray]] = None,
+        normalise_func_kwargs: Optional[Dict[str, ...]] = None,
+        perturb_func: Callable = None,
+        perturb_func_kwargs: Optional[Dict[str, ...]] = None,
+        return_aggregate: bool = False,
+        aggregate_func: Optional[Callable[[np.ndarray], np.float]] = np.mean,
+        disable_warnings: bool = False,
+        display_progressbar: bool = False,
+        eps_min: float = 1e-6,
+        default_plot_func: Optional[Callable] = None,
+        return_nan_when_prediction_changes: bool = True,
+        **kwargs: Dict[str, ...],
+    ):
+        """
+        Parameters
+        ----------
+        nr_samples: int
+            The number of samples iterated, default=200.
+        abs: boolean
+            Indicates whether absolute operation is applied on the attribution.
+        normalise: boolean
+            Flag stating if the attributions should be normalised
+        normalise_func: callable
+            Attribution normalisation function applied in case normalise=True.
+        normalise_func_kwargs: dict
+            Keyword arguments to be passed to normalise_func on call, default={}.
+        perturb_func: callable
+            Input perturbation function. If None, the default value is used, default=gaussian_noise.
+        perturb_func_kwargs: dict
+            Keyword arguments to be passed to perturb_func, default={}.
+        return_aggregate: boolean
+            Indicates if an aggregated score should be computed over all instances.
+        aggregate_func: callable
+            Callable that aggregates the scores given an evaluation call.
+        disable_warnings: boolean
+            Indicates whether the warnings are printed, default=False.
+        display_progressbar: boolean
+            Indicates whether a tqdm-progress-bar is printed, default=False.
+        default_plot_func: callable
+            Callable that plots the metrics result.
+        eps_min: float
+            Small constant to prevent division by 0 in relative_stability_objective, default 1e-6.
+        return_nan_when_prediction_changes: boolean
+            When set to true, the metric will be evaluated to NaN if the prediction changes after the perturbation is applied, default=True.
+        """
+
+        if normalise_func is None:
+            normalise_func = normalise_by_negative
```

Comment:
We have updated the default normalise function to `normalise_by_second_moment_estimate` please update in the 3 metrics

Reply:
I was unable to find a function named `normalise_by_second_moment_estimate`. I think this could have been a typo, and you meant `normalise_by_average_second_moment_estimate`. Moreover, this function is not used in any of the metrics as default 
![image](https://user-images.githubusercontent.com/44274784/207604741-75cdacd1-9802-461f-bbd2-8c278c383814.png)

Reply:
changed to 
```       
 if normalise_func is None:
      normalise_func = normalise_by_average_second_moment_estimate
```

Reply:
Thank you sorry I missed the "average"